test/component/utils/mock_data/home/mock_posts.dart

```diff
@@ -0,0 +1,26 @@
+import "package:proxima/models/ui/post_data.dart";
```

Comment:
We should maybe check corner cases posts  (negative numbers, empty names,...)

Reply:
Yes, but I think that we should that in mind when working on the view model.